plugins/explore/src/components/GroupsExplorerContent/GroupsDiagram.tsx

```diff
@@ -107,7 +108,12 @@ function RenderNode(props: DependencyGraphTypes.RenderNodeProps<any>) {
  * Dynamically generates a diagram of groups registered in the catalog.
  */
 export function GroupsDiagram() {
-  const nodes = new Array<{ id: string; kind: string; name: string }>();
+  const nodes = new Array<{
+    id: string;
+    kind: string;
+    name: string;
+    profile?: any;
```

Comment:
You forgot to remove this line - after that, let's ship it!

Reply:
Alright, thanks for noticing